docs/en/config/_autotune.md

```diff
@@ -127,6 +127,20 @@ Additional notes:
 
 - Whether tuning is applied while flying or after landing can be [configured using parameters](#apply-tuning-when-in-air-landed).
 
+<div v-if="$frontmatter.frame === 'Multicopter'">
+
+## Autotuning Large Vehicles
+
+For big multicopter vehicles you may need to increase the desired raise time of the step response [MC_AT_RISE_TIME](../advanced_config/parameter_reference.md#MC_AT_RISE_TIME).
+This requires some trial and error as an appropriate rise time depends on both vehicle size and the rotor response.
+
+Note that if there are slow oscillations in pretuning it may mean that the attitude loop is too fast compared to the rate loop.
+In this case you should troubleshoot as described in [Drone oscillates when performing the pre-tuning test](#drone-oscillates-when-performing-the-pre-tuning-test).
+
+<!-- Fixed wing rise time is hardcoded (it's lower than on multirotors and is probably enough for most platforms). -->
```

Comment:
@bresch 

This new section is only displayed on MC version of tuning doc

1. Is it correct that we _increase_ MC_AT_RISE_TIME for large vehicles?
2. Would we deal with slow oscillations before autotuning - i.e. are the sentences back to front.
3. Is there anything else we should say?

Should we have sections for FW/VTOL? If so, let me know what they should say. For now I am assuming not needed.

Reply:
> Is it correct that we increase MC_AT_RISE_TIME for large vehicles?

yes

Reply:
Thanks! I'm going to count this as approval and self-merge.